draft-curriculum.md

```diff
@@ -0,0 +1,40 @@
+# Draft Curriculum St. Andrews 2024 #
+
+## Day 1 - Introduction to Bonsai ##
+- Introduction to Bonsai. What is visual reactive programming?
+- Marble diagrams and how to read them
+- Learning your way around the Bonsai IDE
+- Introduction to acquiring signals in Bonsai
+
+Tutorial: acquisition and tracking
+
+## Day 2 - Data synchronisation ##
+- Fundamental reactive operators
+- Logging stimulus and response outcomes
+- Continuous and conditional feedback: closed-loop experiment building blocks
+- Synchronising asynchronous data streams
+
+Tutorial: closed-loop systems, data synchronisation
+
+## Day 3 - Designing behavioral experiments ##
+- Synchronising asynchronous data streams
```

Comment:
Just noticed this is repeated from the previous day, is the idea to do a recap or pickup any missing bits?